src/ripple/protocol/impl/STVar.h

```diff
@@ -66,12 +73,12 @@ class STVar
 
     STVar(STBase&& t)
     {
-        p_ = t.move(max_size, &d_);
+        p_ = t.move(sizeof(d_), &d_);
```

Comment:
Is there any advantage to using the `sizeof` function instead of a `static` variable?
The existing design uses `static constexpr` which caches the size of the buffer. I suspect the compiler performs such an optimization in your code too.

Reply:
I believe using `sizeof` is less error prone and communicates the intent better. It may _also_ help reduce false positive warnings by the compiler, though that's a bit of a stretch.